Shared/CoffAppApp.swift

```diff
@@ -14,7 +13,6 @@ struct CoffAppApp: App {
     var body: some Scene {
         WindowGroup {
             ContentView()
-                .environment(\.managedObjectContext, persistenceController.container.viewContext)
```

Comment:
I think I'm missing how the environment changed here. Was this just not used anywhere? Why are we able to remove this line?

Reply:
@rodericj It wasn’t used anywhere. Just template cruft.